scripts/katello_remove.sh

```diff
@@ -8,11 +8,26 @@ then
     exit 1
 fi
 
-katello-service stop
+service elasticsearch stop
+service foreman stop
+service foreman-proxy stop
+service httpd stop
+service katello-jobs stop
+service mongod stop
+service postgresql stop
+service pulp_celerybeat stop
+service puppet stop
+service qpidd stop
+service tomcat6 stop
```

Comment:
This will need to be conditional - tomcat6 for EL6 and tomcat for EL7 and Fedora.